parent/pom.xml

```diff
@@ -1237,6 +1237,40 @@
                 <artifactId>elasticsearch-maven-plugin</artifactId>
                 <version>${version.elasticsearch.plugin}</version>
              </plugin>
+             <plugin>
+                <groupId>org.apache.karaf.tooling</groupId>
+                <artifactId>karaf-maven-plugin</artifactId>
+                <version>${version.karaf-maven-plugin}</version>
+                <configuration>
+                   <descriptors>
+                      <descriptor>file://${project.build.directory}/classes/features.xml</descriptor>
+                      <!-- ideally, those profiles should be referenced in the features file -->
```

Comment:
Those profiles are the `<descriptor>` elements below, or something in them? Like I said, I don't know much about osgi :)